update-help-messages.sh

```diff
@@ -0,0 +1,129 @@
+#! /bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# This script is used to update the help messages in the docs.
+
+INCLUDE_DIR="docs/include"
+
+# Given a single command, print the help text, and wrap the output in rst style
+# code block format in a file under include directory
+function print_help_to_file() {
+
+
+    # expand all positional parameters and trim whitespace at the end
+    local cmd
+    cmd=$(echo "$*" | sed -E "s/^ +//")
+
+    # Replace spaces in the file name with dashes We print the output of
+    # `pgcopydb --help` to `pgcopydb.rst`
+    # `pgcopydb compare --help` to `compare.rst`
+    # `pgcopydb compare data --help` to `compare-data.rst`
+    local file_path
+    local help_cmd
+    if [ "${cmd}" = "" ]; then
+        help_cmd="pgcopydb --help 2>&1"
+        file_path="${INCLUDE_DIR}/pgcopydb.rst"
+    else
+        help_cmd="pgcopydb ${cmd} --help 2>&1"
+        file_path="${INCLUDE_DIR}/${cmd// /_}.rst"
+    fi
```

Comment:
Can we transform the existing dashes and double them, and then replace the space with a dash again?